mac/Keyman4MacIM/Keyman4MacIM/Privacy/PrivacyConsent.m

```diff
@@ -1,4 +1,4 @@
-/**
+/*
```

Comment:
was it deliberate to remove the /**

Reply:
Yes, @mcdurdin said that we should avoid java-doc style comments for the header but elsewhere in the file is ok.

Reply:
Yep, this is in the style guide.